pkgs/by-name/op/opencode/package.nix

```diff
@@ -44,6 +44,7 @@ stdenvNoCC.mkDerivation (finalAttrs: {
 
     ldflags = [
       "-s"
+      "-w"
```

Comment:
I think that it is not needed because `-s` implies `-w`:
https://tip.golang.org/doc/go1.22#linker